paymentsheet/src/main/java/com/stripe/android/paymentelement/confirmation/DefaultConfirmationHandler.kt

```diff
@@ -142,6 +145,7 @@ internal class DefaultConfirmationHandler(
         when (val action = mediator.action(confirmationOption, intent)) {
             is ConfirmationMediator.Action.Launch -> {
                 storeIsAwaitingForResult(
+                    key = mediator.key,
```

Comment:
I imagine we never will have a case like this but if we ever decide that the same definition can be looped through multiple times in a confirmation flow, this will break. Other option to use UUID instead.

Reply:
Feels like a fine trade off for now 👍